Sofa/Sofa/Database/DatabaseManager.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  DatabaseManager.swift
+//  Sofa
+//
+//  Created by Akademija on 16.04.2025..
+//
+
+import RealmSwift
+
+class DatabaseManager {
```

Comment:
DataManager bi trebao imati samo osnovne CRUD operacije u sebi, nad jednim entitetom. 
Znači za event, save, update, delete...

Ono što tebi treba je još jedna apstrakcija iznad. izvana kad radiš nešto za bazom sve radiš preko StorageManager (može se i drugačije zvati) koji wrappa u sebi DatabaseManager, i poziva te CRUD metode. 

I u njemu zapravo handlaš array i množinu... save all events, deleteAll...

Reply:
Tako se ispravno radi apstrakcija nad bazom.